dashboard/app/models/script.rb

```diff
@@ -988,7 +988,8 @@ def clone_with_suffix(new_suffix, options = {})
     new_properties = {
       is_stable: false,
       tts: false,
-      script_announcements: nil
+      script_announcements: nil,
+      is_course: false
```

Comment:
Added this because of a unit test failure; it was trying to create a `CourseVersion` for the copy, but it had the same CourseOffering key + CourseVersion key. I think this might make sense, though. If we want the new copy to be hidden initially, it seems like it would make sense to not create a CourseOffering for it yet. Thoughts?